extensions/survey/renderer.tsx

```diff
@@ -8,6 +8,7 @@ export default class SurveyRendererExtension extends LensRendererExtension {
   appPreferences = [
     {
       title: "In-App Surveys",
+      showInPreferencesTab: "telemetry",
```

Comment:
Telemetry extensions are meant to be moved in separate tab away from other ones.